test/e2e/page-objects/pages/send/solana-send-page.ts

```diff
@@ -29,13 +34,55 @@ class SendSolanaPage {
     this.driver = driver;
   }
 
-  async clickOnSwapCurrencyButton(): Promise<void> {
-    await this.driver.waitForControllersLoaded();
-    const swapCurrencyButton = await this.driver.waitForSelector(
-      this.swapCurrencyButton,
-      { timeout: 10000 },
+  async check_amountCurrencyIsDisplayed(currency: string): Promise<void> {
+    await this.driver.waitForSelector(this.amountCurrencyLabel(currency));
+  }
+
+  async check_pageIsLoaded(amount: string = '') {
+    await this.driver.waitForSelector(this.toAddressInput, { timeout: 10000 });
+    console.log('check_pageIsLoaded after waitForSelector');
+    if (amount) {
+      await this.driver.waitForSelector(
+        {
+          text: `${amount}`,
+          tag: 'p',
+        },
+        { timeout: 60000 },
+      );
+    }
+    await this.driver.delay(1000); // Added because of https://consensyssoftware.atlassian.net/browse/SOL-116
+  }
+
+  async check_tokenBalanceIsDisplayed(
+    amount: string,
+    tokenName: string,
+  ): Promise<void> {
+    await this.driver.clickElement({
+      text: `Balance: ${amount} ${tokenName}`,
+      tag: 'p',
+    });
+  }
+
+  async check_tokenByNameIsDisplayed(tokenName: string): Promise<void> {
+    await this.driver.waitForSelector(
+      {
+        text: tokenName,
+        tag: 'p',
+      },
+      { timeout: 2000 },
+    );
+  }
+
+  async check_TxSimulationFailed(): Promise<void> {
+    await this.driver.waitForSelector(
+      { text: 'Transaction simulation failed', tag: 'p' },
+      { timeout: 5000 },
+    );
+    await this.driver.waitForSelector(
+      { text: 'This transaction was reverted during simulation.', tag: 'p' },
+      { timeout: 5000 },
     );
-    await swapCurrencyButton.click();
```

Comment:
we should use the driver clickElement method instead, which it has all the guards to prevent flakiness